ponyo/helper_vae.py

```diff
@@ -0,0 +1,131 @@
+'''
+Author: Alexandra Lee
+Date Created: 30 August 2019
+
+Helper functions for running variational autoencoder `vae.py`
+'''
+
+import os
+import argparse
+import pandas as pd
+import matplotlib.pyplot as plt
+import tensorflow as tf
+
+# To ensure reproducibility using Keras during development
+# https://keras.io/getting-started/faq/#how-can-i-obtain-reproducible-results-using-keras-during-development
```

Comment:
You use the logic from this link in your code, but the comment itself is separated from the code it refers to